docker/vast/docker-compose.web.yaml

```diff
@@ -0,0 +1,10 @@
+version: "3"
+services:
+  vast:
+    environment:
+      - VAST_START__COMMANDS=web server
```

Comment:
is there a doc somewhere about how the web server is configured / started?

Reply:
Not yet, but we have a separate story for writing documentation for the web plugin. That should not be part of this PR.